scpdt/plugin.py

```diff
@@ -159,6 +162,58 @@ def collect(self):
 
 def _get_parser():
     """
-    Return instance of DTParser with default configuration
+    Return instance of DTParser
+    """
+    return DTParser(config=dt_config)
+
+
+def _get_runner(checker, verbose, optionflags):
+    import doctest
```

Comment:
Nit: there is probably no reason to hide imports; move it to the module level.

Reply:
It's because of this error: 
```python
AttributeError: module '_pytest.doctest' has no attribute 'UnexpectedException'
```